kratos/utilities/particles_utilities.h

```diff
@@ -252,17 +252,18 @@ class ParticlesUtilities
         typename BinBasedFastPointLocator<TDim>::ResultContainerType TLS(max_results);
 
         auto interpolations = std::make_pair(std::vector<bool>(rCoordinates.size1()), std::vector<TDataType>(rCoordinates.size1()));
+        std::vector<char> is_found_aux_vector(rCoordinates.size1()); //aux vector using ints to allow running in omp
```

Comment:
let's return DenseVector<bool>

this is a stupid mistake from my side. Sorry about this